src/beam.ts

```diff
@@ -29,7 +28,7 @@ function calculateStemDirection(notes: StemmableNote[]) {
   return Stem.UP;
 }
 
-function getStemSlope(firstNote: StemmableNote, lastNote: StemmableNote) {
+function getStemSlope(firstNote: Note, lastNote: Note) {
```

Comment:
I'm trying to understand the rationale here -- it feels like we're replacing compile-time checks with runtime checks. It's good to have `StemmableNote` a parameter here (you can't beam notes are not stemmable.)